### PR TITLE
Switch from lazy_static to once_cell

### PR DESCRIPTION
**Pull request by tranzystorek-io**

### CODE REVIEW

File: src/unix.rs [hunk 1]
```diff
@@ -6,27 +6,24 @@
...
 use crate::constants::POINTER_SIZE_CONSERVATIVE;
 
 const UPPER_BOUND_COMMAND_LINE_LENGTH: i64 = 16 * 1024 * 1024;
 
-lazy_static! {
-    static ref PAGE_SIZE: i64 = {
-        sysconf(SysconfVar::PAGE_SIZE)
-            .ok()
-            .flatten()
-            .map(|page_size| page_size as i64)
```
Comment (sharkdp) [inline src/unix.rs:21]: The cast is still needed for 32bit platforms. We should probably add a comment why it's there.

**Comment by sharkdp** [comment]
Thank you. This requires bumping MSRV. That's okay, but let's be conservative and only bump as much as required.

File: src/unix.rs (hunk 1 above)
Comment (tranzystorek-io) [reply src/unix.rs:21]: Done, sorry, should have asked before removing

**Comment by tranzystorek-io** [comment]
I'm running cargo msrv to see what it says

**Comment by sharkdp** [comment]
Thank you. Didn't know about `cargo msrv`!

**Pull request merged**